### ISSUE TITLE
"Create new Wallet" button label

### ISSUE THREAD

**Issue by anonymoussprocket** [Cryptonomic/Tezori#488]
On the Create New Wallet view, the file selector button should be labeled as "Select file", not "Create New Wallet".

**Comment by edwinportillo21** [comment]
I am not able to create a new wallet and I am a new user. Only select file is showing.

**Comment by anonymoussprocket** [comment]
Yes, click that button, it simply lets you select the destination for the new file
